.github/workflows/contracts.yaml

```
@@ -113,29 +113,19 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Get upstream packages versions
-        uses: keep-network/ci/actions/upstream-builds-query@v1
-        id: upstream-builds-query
-        with:
-          upstream-builds: ${{ github.event.inputs.upstream_builds }}
-          query: keep-core-contracts-version = github.com/keep-network/keep-core/solidity-v1#version
-
       - name: Resolve latest contracts
-        run: |
-          yarn upgrade \
-            @keep-network/keep-core@${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }}
+        run: yarn upgrade @keep-network/keep-core@1.8.1-goerli.0
```

Could we get a package that is tagged with the environment name?

```suggestion
        run: yarn upgrade @keep-network/keep-core@${{ github.event.inputs.environment }}
```

---

Yes, we can. Resolving `keep-core` dependency using `goerli` tag did not work for me in another project, but here apparently it works fine. Changed in c9f33ca2306bf8298a97d9981540c9544f172393.